lib/screens/headways.ex

```diff
@@ -117,16 +117,18 @@ defmodule Screens.Headways do
     },
     red_ashmont: {~w[Red], ~w[shmnl fldcr smmnl asmnl]},
     red_braintree: {~w[Red], ~w[nqncy wlsta qnctr qamnl brntn]},
-    silver_seaport: {~w[741 742 746], ~w[conrd wtcst crtst sstat]},
-    silver_chelsea: {~w[743], ~w[conrd wtcst crtst sstat aport estav boxdt belsq chels]}
+    # combining 743 with the other three for the common stops
+    silver_seaport: {~w[741 742 743 746], ~w[conrd wtcst crtst sstat]},
```

Comment:
Wanted to bring up this change - it looks like on PA/ESS WTC is configured to be just `silver_seaport` regardless of it's the 743 or not. I combined all of the "trunk"-ish stops, but I can split them out again if that doesn't feel quite right.